infra/docker-compose/docker-compose.yml

```diff
@@ -18,10 +18,30 @@ services:
       - /opt/feast/feast-core.jar
       - --spring.config.location=classpath:/application.yml,file:/etc/feast/application.yml
 
+  jobservice:
+    image: gcr.io/kf-feast/feast-jobservice:${FEAST_VERSION}
+    depends_on:
+      - core
+    ports:
+      - 6568:6568
+    volumes:
+      - $PWD:/shared
+    environment:
+      FEAST_CORE_URL: core:6565
+      FEAST_SERVING_URL: online_serving:6566
+      FEAST_SPARK_LAUNCHER: standalone
+      FEAST_SPARK_STANDALONE_MASTER: local
+      FEAST_SPARK_HOME: /usr/local/spark
+      FEAST_SPARK_STAGING_LOCATION: file:///shared/staging
+      FEAST_HISTORICAL_FEATURE_OUTPUT_LOCATION: file:///shared/historical_feature_output
+      FEAST_HISTORICAL_FEATURE_OUTPUT_FORMAT: parquet
+      FEAST_REDIS_HOST: redis
```

Comment:
Do we need a port?

Reply:
`FEAST_REDIS_PORT` is a separate options, and its default value `6379` was not changed in this case. So I believe it was omitted to save 1 line)